src/components/table/columns.ts

```diff
@@ -34,6 +38,27 @@ export class ColumnDefinitionFactory {
     }
 }
 
+export const formatTitle = (column: Column) => (): string | HTMLElement => {
+    const element = document.createElement('div');
+    element.innerText = column.title;
+
+    const icon = document.createElement('limel-icon');
+    icon.setAttribute('name', column.menuIcon);
+    icon.setAttribute('size', 'x-small');
+    icon.setAttribute('class', 'tabulator-header-menu-button');
+    icon.setAttribute('style', 'padding: 0 0 0 5px;');
```

Comment:
The styling part will be handled by @Kiarokh

Reply:
We should probably do it in the CSS instead of inline :slightly_smiling_face:

Reply:
yes, I'll put all the styles in a CSS file. You can remove the inline styles. Put them in the respective file for now just as a reference, and I'll tweak them.